src/unlimited.h

```diff
@@ -186,32 +199,33 @@ extern std::set<uint256> setUnVerifiedOrphanTxHash;
 extern CCriticalSection cs_xval;
 // Xpress Validation: end
 
-extern void LoadFilter(CNode *pfrom, CBloomFilter *filter);
+extern void LoadFilter(CNode* pfrom, CBloomFilter* filter);
 
```

Comment:
Per the review for the unlimited.cpp, then this also would need to be changed to have the star on the other side.